lib/msal-common/src/client/SilentFlowClient.ts

```diff
@@ -71,9 +72,11 @@ export class SilentFlowClient extends BaseClient {
                 refreshToken: cachedRefreshToken.secret
             };
 
-            return refreshTokenClient.acquireToken(refreshTokenRequest);
+            return refreshTokenClient.acquireToken(refreshTokenRequest, telemetryManager);
         }
 
+        // Return tokens from cache
+        telemetryManager.incrementCacheHits();
```

Comment:
I believe cache hits should only be incremented for access token hits, not when the tokens have to be refreshed.

Reply:
That's what this is doing, if the token is refreshed it is returned above this line

Reply:
Oh I see, thanks for clarifying.